ruff_lsp/server.py

```diff
@@ -121,7 +121,7 @@ class VersionModified(NamedTuple):
     "--no-cache",
     "--no-fix",
     "--quiet",
-    "--format",
+    "--output-format",
```

Comment:
I think we need to do this in a backwards-compatible way, otherwise this will fail on all versions of Ruff prior to when that flag was introduced. We probably need to vary these arguments depending on the detected Ruff version?

Reply:
Yeah I went to bump the versions then realized backwards compatibility was probably better — I did this as you were reviewing :)